modules/math/shared/src/main/scala/gsp/math/optics/SplitEpi.scala

```diff
@@ -22,6 +23,14 @@ import monocle.{ Iso, Prism }
  */
 final case class SplitEpi[A, B](get: A => B, reverseGet: B => A) {
 
+  /** Modify the target of the SplitEpi using a function. */
+  def modify(f: B => B): A => A =
+    a => reverseGet(f(get(a)))
+
+  /** Modify the target of the SplitEpi using Functor. */
+  def modifyF[F[_]: Functor](f: B => F[B])(a: A): F[A] =
```

Comment:
It seems inconsistent to have these methods curried differently but that's how Monocle does it do it seems fine to me.

Reply:
Right, I thought it was odd too but I decided it would be best to be consistent with Monocle.